torch/utils/data/datapipes/iter/readfilesfromtar.py

```diff
@@ -0,0 +1,60 @@
+from torch.utils.data.dataset import IterableDataset as IterDataPipe
```

Comment:
I find an easy way just by aliasing `IterableDataset` as `IterDataPipe` in the `__init__.py` under data, then directly import IterDataPipe.

Reply:
PR #51488

Reply:
Great! Updated